compose/foundation/foundation/src/desktopMain/kotlin/androidx/compose/foundation/BasicContextMenuRepresentation.desktop.kt

```diff
@@ -87,17 +88,17 @@ class DefaultContextMenuRepresentation(
                 popupPositionProvider = rememberCursorPositionProvider(),
                 onKeyEvent = {
                     if (it.type == KeyEventType.KeyDown) {
-                        when (it.key) {
-                            Key.Escape -> {
+                        when (it.key.nativeKeyCode) {
+                            java.awt.event.KeyEvent.VK_ESCAPE -> {
```

Comment:
Check for the nativeKeyCode directly without using newly added API, because the new API is internal in another module - material. 
In this case, there is no need in having similar common API in this module, because this feature is only on desktop for now.